flink-runtime-web/web-dashboard/src/app/components/humanize-watermark.pipe.ts

```diff
@@ -44,8 +44,8 @@ export class HumanizeWatermarkToDatetimePipe implements PipeTransform {
   constructor(private readonly configService: ConfigService) {}
 
   public transform(value: number): number | string {
-    if (isNaN(value) || value <= this.configService.LONG_MIN_VALUE) {
-      return '-';
+    if (value == null || isNaN(value) || value <= this.configService.LONG_MIN_VALUE) {
+      return 'N/A';
```

Comment:
Why we introduce these changes in this PR?

Reply:
We'll humanize the timestamps of `ProfilingStartTime` or `ProfilingFinishedTime` on the web. 
While `isNan(null) =false` when `value` is null, this caused the parse-out time to be wrong. So changes have been made in this PR to make this PipeTransform more generic.